packages/rspack/src/config/zod/resolve.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+
+const resolveAlias = z.record(
+	z
+		.literal(false)
+		.or(z.string())
+		.or(z.array(z.literal(false).or(z.string())))
+);
+
+export function resolve(): z.ZodSchema<Record<string, unknown>> {
```

Comment:
```suggestion
export function resolve() {
```

You don't need to specify the type.

Reply:
It seems like must to have a return type because reference itself in `z.lazy`

Reply:
> Get it. I'll take it from here. Thank you.

Ok. This is my first pr. Thank you for your guidance 🤣